lib/instrumentation/langchain/common.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const {
+  AI: { LANGCHAIN }
+} = require('../../metrics/names')
+
+const common = module.exports
+
+common.getTags = function getTags(localTags = [], paramsTags = []) {
```

Comment:
just FYI this file has changed in my [PR](https://github.com/newrelic/node-newrelic/pull/2009/files#diff-703fcaaae940d9345dbd60dd453358c8a1a7e9a841ab87a4bb09a83df75fe9aaR4)

Reply:
Ok. I'll update it on my end as well.